conda/recipes/rapids-doc-env/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - pip
   run:
     - beautifulsoup4
-    - doxygen
+    - doxygen=1.8.20
```

Comment:
Version control is determined with the `versions.yml` file under `conda/recipes`.

Reply:
Ah.. missed that one. Thanks. Fixed it.